src/rascal/structure_managers/property_typed.hh

```diff
@@ -319,7 +319,11 @@ namespace rascal {
      * Accessor for last pushed entry for dynamically sized properties
      */
     reference back() {
-      auto && index{this->values.size() - this->get_nb_comp()};
+      // -1 is correction for 0-start indexing
+      auto && index{this->values.size() / this->get_nb_comp() - 1};
+      if (index < 0) {
```

Comment:
if `this->values.size()` is 0, index will be -1/SIZE_MAX. So this check will not catch this case. Is it checked elsewhere?

Reply:
I'll switch the runtime check around and just test for an empty property.